pkg/controller/controller.go

```diff
@@ -671,6 +671,8 @@ func (c *Controller) processItem(newEvent Event) error {
 		substring := strings.Split(newEvent.key, "/")
 		newEvent.namespace = substring[0]
 		newEvent.key = substring[1]
+	} else {
+		newEvent.namespace = objectMeta.Namespace
```

Comment:
When will we get here?

Reply:
I just added it as a fallback to the old behavior